libs/cases/frontend/view/src/lib/components/open-cases/open-cases.component.scss

```diff
@@ -0,0 +1,11 @@
+.table {
+	color: #777784;
+	& .emergency {
+		background-color: #ff26401a;
+		color: #ffffffbf;
+	}
+
+	& .status-open {
+		color: #ffffffbf;
```

Comment:
is it possible to add the status-open class to elements of emergency as well? Then you dont need to add the color attribute again (Dont Repeat Yourserlf)

Reply:
Can try.

But with the note from above on exporting the scss to a lib I would just paste that in.